### PR TITLE
docs: DOC-232: New LLM templates and reorganizing

### PR DESCRIPTION
**Pull request by caitlinwheeless**
Renaming and restructuring the GenAI templates section, plus adding 5 new templates. 

This change affects:

- [X] Community docs
- [X] Enterprise docs

### CODE REVIEW

File: docs/source/templates/llm_ragas.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+* An LLM-generated response to the prompt (the ML backend example uses OpenAI). 
+* A Ragas score for faithfulness and accuracy.
+* An LLM-generated evaluation of the response. 
+* A comprehensive overview of precisely which documents were used for context. 
+
+## Prerequisites
+
+This template requires an ML backend to work. Follow the instructions outlined in [RAG Quickstart Labeling](https://github.com/HumanSignal/label-studio-ml-backend/tree/master/label_studio_ml/examples/rag_quickstart) to connect the ML backend to your project. 
+
```
Comment (caitlinwheeless) [inline docs/source/templates/llm_ragas.md:24]: This link is broken for now - should work once that example is merged into master.

File: docs/source/templates/llm_ragas.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+meta_description: Use Ragas metrics to evaluation LLM responses. 
+date: 2024-07-26 14:49:57
+---
+
+<img src="/images/templates/evaluate-rag-automated-metrics.png" alt="" class="gif-border" width="700px"/>
+
+This template uses the [Ragas](https://docs.ragas.io/en/stable/) framework to evaluate your RAG pipeline. When given a prompt, it will use Ragas and OpenAI to return the following:
+
+* An LLM-generated response to the prompt (the ML backend example uses OpenAI). 
+* A Ragas score for faithfulness and accuracy.
```
Comment (jombooth) [inline docs/source/templates/llm_ragas.md]:
```suggestion
* Ragas scores for [faithfulness](https://docs.ragas.io/en/latest/concepts/metrics/faithfulness.html) and [answer relevancy](https://docs.ragas.io/en/latest/concepts/metrics/answer_relevance.html).
```

**Review by jombooth — APPROVED** [review]
LGTM for rag_quickstart changes!

File: docs/source/templates/llm_ragas.md (hunk 1 above)
Comment (jombooth) [reply docs/source/templates/llm_ragas.md:24]: Can we actually link the version on the feature branch for the time being? https://github.com/HumanSignal/label-studio-ml-backend/tree/agi-builders-workshop-rag/label_studio_ml/examples/rag_quickstart

**Pull request merged**